packages/grid/src/vaadin-grid.js

```diff
@@ -911,16 +911,15 @@ class Grid extends ElementMixin(
 
   /** @private */
   _updateScrollerItem(row, index) {
+    row._index = index;
```

Comment:
A row element already has `index` property. Can that be used instead?

Reply:
Thanks, done.